WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -65,6 +68,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
 
         ThemeViewHolder themeViewHolder = new ThemeViewHolder(view);
         view.setTag(themeViewHolder);
+        mViewWidth = parent.getWidth();
```

Comment:
That will work great for singlecolums, but we'll have  for multicolumn, `parent.getWidth()` is probably not what we want (resolution will be huge compared to what we're displaying).